### PR TITLE
fix(components/inline-form): remove inline form race condition

### PR DESCRIPTION
**Pull request by bradenbiz**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/670/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/670/storybooks/ag-grid-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/670/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/670/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/670/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/670/playground/)

File: libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts [hunk 1]
```diff
@@ -132,152 +137,130 @@ export class SkyInlineFormComponent implements OnInit, OnDestroy {
...
-                  {
-                    text: values[1],
-                    styleType: 'link',
-                    action: 'cancel',
-                  },
-                ]);
-              }
-            );
-          });
+        ).subscribe((values: string[]) => {
```
Comment (Blackbaud-SteveBrush) [inline libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts]: Does this subscription need a `takeUntil(this.#ngUnsubscribe)`?

File: libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts [hunk 2]
```diff
@@ -132,152 +137,130 @@ export class SkyInlineFormComponent implements OnInit, OnDestroy {
...
```
Comment (Blackbaud-SteveBrush) [inline libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts]: Does this subscription need a `takeUntil(this.#ngUnsubscribe)`?

File: libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts [hunk 3]
```diff
@@ -132,152 +137,130 @@ export class SkyInlineFormComponent implements OnInit, OnDestroy {
...
-                  {
-                    text: values[2],
-                    styleType: 'link',
-                    action: 'cancel',
-                  },
-                ]);
-              }
-            );
-          });
+        ).subscribe((values: string[]) => {
```
Comment (Blackbaud-SteveBrush) [inline libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts]: Does this subscription need a `takeUntil(this.#ngUnsubscribe)`?

File: libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts [hunk 4]
```diff
@@ -132,152 +137,130 @@ export class SkyInlineFormComponent implements OnInit, OnDestroy {
...
```
Comment (Blackbaud-SteveBrush) [inline libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts]: Does this subscription need a `takeUntil(this.#ngUnsubscribe)`?

File: libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts (hunk 1 above)
Comment (bradenbiz) [reply libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts]: I do that on line 150 when `#getPresetButtons` is called. Is that sufficient?

Comment (Blackbaud-SteveBrush) [reply libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts]: That `takeUntil` is for the `emitter` subscription, not for the `getString` subscriptions.

File: libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts (hunk 4 above)
Comment (bradenbiz) [reply libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts]: done in [pipe takeUntil unsubscribe on resource string subscriptions](https://github.com/blackbaud/skyux/pull/670/commits/d720487ef67be7bf88df3aba4ae495d50b5ee070)

File: libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts (hunk 3 above)
Comment (bradenbiz) [reply libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts]: done in: [pipe takeUntil unsubscribe on resource string subscriptions](https://github.com/blackbaud/skyux/pull/670/commits/d720487ef67be7bf88df3aba4ae495d50b5ee070)

File: libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts (hunk 2 above)
Comment (bradenbiz) [reply libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts]: done in: [pipe takeUntil unsubscribe on resource string subscriptions](https://github.com/blackbaud/skyux/pull/670/commits/d720487ef67be7bf88df3aba4ae495d50b5ee070)

File: libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts (hunk 1 above)
Comment (bradenbiz) [reply libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts]: done in: [pipe takeUntil unsubscribe on resource string subscriptions](https://github.com/blackbaud/skyux/pull/670/commits/d720487ef67be7bf88df3aba4ae495d50b5ee070)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**